src/esn.calendar.libs/app/components/event-preview-popover/event-preview-popover.pug

```diff
@@ -37,14 +37,14 @@
       span {{ $ctrl.attendees.length }} {{ 'attendees' | translate }}
       span.secondary-description-text {{ $ctrl.attendees | attendeePartStatText }}
       .attendee-list
-        .attendee-item(ng-repeat="attendee in $ctrl.attendees | limitTo: 5")
+        .attendee-item( ng-repeat="attendee in $ctrl.attendees | limitTo: $ctrl.limit track by $id(attendee)")
           cal-attendee-avatar(attendee='attendee')
           .attendee-item-meta
             .name
               span {{ ::attendee.displayName }}
             .details(ng-if="attendee.email === $ctrl.event.organizer.email")
               span.organizer {{ 'Organizer' | translate }}
-      span.secondary-description-text(ng-if="$ctrl.attendees.length > 5") {{ $ctrl.attendees.length - 5 }} {{ 'more attendees...' | translate }}
+        a.clickable( ng-if="$ctrl.showToggle()", ng-href="", ng-click="$ctrl.showAll()") {{ 'Show all...' | translate }}
```

Comment:
```suggestion
        a.clickable(ng-if="$ctrl.showToggle()", ng-href="", ng-click="$ctrl.showAll()") {{ 'Show all...' | translate }}
```
also 
`Show all...` should be translated I guess, or you can just keep the old `more attendees...`

Reply:
it is already translated because i used the same message as edit event